.circleci/config.yml

```diff
@@ -0,0 +1,19 @@
+version: 2.1
+jobs:
+  build:
+    machine:
+      # https://circleci.com/docs/2.0/configuration-reference/#available-machine-images
+      image: ubuntu-1604:201903-01
+
+    working_directory: ~/akka-safer-serializer
+
+    steps:
+      - checkout
+
+      # -batch is necessary to make sure sbt doesn't stall waiting for user input in case of certain failures
+      - run: echo '-batch' >> .sbtopts
+      - run: sbt update
+      - run: sbt +scalafmtCheckAll
```

Comment:
Actually, so far as we're NOT cross-building yet (we're only targetting Scala 2.12, and not e.g. 2.13), the `+`s have no effect, you can remove them

Reply:
(#9 for 2.13 support)